doc/airline.txt

```diff
@@ -1216,6 +1216,30 @@ Note: Not displayed if the number of tabs is less than 1
 
   let g:airline#extensions#tabline#formatter = 'short_path'
 
+* defines the customized format() function to display tab title in tab mode. >
+  let g:airline#extensions#tabline#tabtitle_formatter = 'MyTabTitleFormatter'
+<
+  Then define the MyTabTitleFormatter() function in your vimrc. >
+    function MyTabTitleFormatter(n)
```

Comment:
Is there a prebuilt formatter for this custom formatter.
This solution is working fine for me but ideally I would like to use a combination of unique_tail + show active split.